web-registry/src/components/atoms/SanityButton.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```

Comment:
I built this component to help with one of the things i've noticed with our sanity setup. We have a `onBtnClick` helper which properly manages modal behavior (though it requires a callback, and sometimes buttons are just for links) and `target` - but because that function has no access to `react-router`'s `navigate` function, internal links won't take advantage of the SPA nature of the app (the whole page will re-load). 

I thought about modifying or creating a new helper function to accept `navigate` as an optional param, but think this component makes more sense overall and should simplify some of the boilerplate for adding buttons from sanity. 

The downside is that this won't work in `web-www`. I tried to make something more flexible (which accepted a `navigate` function through props instead of invoking the `useNavigate` hook so we could pass Gatsby's [navigate](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/#how-to-use-the-navigate-helper-function) and use in `web-www`) but it got a bit tricky with types. 

I *think* I might know how to make something like that work through generics... you'd have a `GenericSanityButton` which would live in `web-components` that accepted `navigate` as a prop and was manually passed the button props as a type variable, and invoke it within both `web-registry` and `web-www`. So in registry you'd have something like (pseudo code):

```tsx
import { useNavigate } from 'react-router-dom'
import GenericSanityButton, { GenericSanityButtonProps } from 'web-components'

type Props = Omit<GenericSanityButtonPops, 'navigate'>

function RegistrySanityButton(props: Props) {
  const navigate = useNavigate()
  return (
    <GenericSanityButton {...props} navigate={navigate} />
  )
}
```

....and also have something similar in `web-www`, but I didn't want to go too much outside of the scope of the story